src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -122,6 +122,29 @@ public FeedbackSessionResultsBundle(FeedbackSessionAttributes feedbackSession,
         // roster.*Table is populated using the CourseRoster data directly
         this.rosterTeamNameMembersTable = getTeamNameToEmailsTableFromRoster(roster);
         this.rosterSectionTeamNameTable = getSectionToTeamNamesFromRoster(roster);
+        if (!areQuestionNumbersConsistent()) {
+            log.severe(feedbackSession.getIdentificationString() + " has invalid question numbers");
+        }
+    }
+    
+
+    //TODO can be removed once we are sure that question numbers will be consistent
+    private boolean areQuestionNumbersConsistent() {
+        Set<Integer> questionNumbersInSession = new HashSet<>();
+        for (FeedbackQuestionAttributes question : this.questions.values()) {
```

Comment:
I am not sure but do the "question"s need to be in order? like if I printed out the question numbers in the loop should they be in order?